docs/inputs/config_file.md

```diff
@@ -54,3 +54,78 @@ outputs:
 
 There are many other config keys also available.
 See the [configuration docs](../config/overview.md) for more details.
+
+### Install IDE Validation
+
+You can validate your `.rich-codex.yml` files in your IDEs using JSONSchema.
+
+#### VSCode
+
+1. Install the [VSCode-YAML extension](https://marketplace.visualstudio.com/items?itemName=redhat.vscode-yaml)
+
+2. In your repo, create a `.vscode/settings.jsonc` or `.vscode/settings.template.jsonc` file containing the following data. This is what tells the extension which schema to associate with each file.
```

Comment:
We should add this to https://www.schemastore.org/json/ so that it loads automatically for any VSCode users.